packages/components/src/templates/next/components/internal/Navbar/NavItem.tsx

```diff
@@ -0,0 +1,123 @@
+import { forwardRef } from "react"
+import { FocusScope } from "react-aria"
+import { BiChevronDown, BiRightArrowAlt, BiX } from "react-icons/bi"
+import { tv } from "tailwind-variants"
+
+import type {
+  NavbarItem as BaseNavbarItemProps,
+  NavbarProps,
+} from "~/interfaces/internal/Navbar"
+
+interface NavbarItemProps
+  extends BaseNavbarItemProps,
+    Pick<NavbarProps, "LinkComponent"> {
+  isOpen: boolean
+  onClick: () => void
+  onCloseMegamenu: () => void
+  megaMenuRef: React.RefObject<HTMLDivElement>
+}
+
+const navbarItemStyles = tv({
+  slots: {
+    megamenu: "absolute left-0 right-0 top-[100%] z-20 bg-white shadow-md",
+    item: "prose-label-md-medium flex h-[4.25rem] flex-row items-center gap-0.5 border-b-2 border-transparent text-base-content-strong transition-all hover:text-brand-interaction-hover motion-reduce:transition-none",
```

Comment:
nit: there is a very small transition from black outline to blue outline when tabbing

https://github.com/user-attachments/assets/2a4dafea-78c6-47b2-943b-4d1367cf2a49

I slow-mo it for you:

https://github.com/user-attachments/assets/718ea8d1-e470-49ee-b52b-79ff27b38049

Reply:
this one due to transition property. actually don't know why lol i set `transition-colors` instead 😆